azure-pipelines.yml

```diff
@@ -116,3 +116,66 @@ jobs:
       fi
     displayName: 'Running benchmarks'
     condition: true
+
+- job: 'Docs'
+  pool:
+    vmImage: ubuntu-16.04
+  timeoutInMinutes: 90
+  steps:
+  - script: |
+      echo '##vso[task.setvariable variable=CONDA_ENV]pandas-dev'
```

Comment:
what is this used for? (I don't find it anywhere used in our CI config)

Reply:
This is used in `ci/setup_env.sh` to create the conda environment. It's standard in all builds.

Reply:
I don't see any "CONDA_ENV" in https://github.com/pandas-dev/pandas/blob/master/ci/setup_env.sh, it seems this name is hardcoded everywhre (also in this yaml file a few lines below)

Reply:
I think you're right, I think it was removed when `ci/setup.sh` was refactored to be used everywhere and is not needed anymore.